yarn-project/circuits.js/package.json

```diff
@@ -22,11 +22,8 @@
     "clean": "rm -rf ./dest .tsbuildinfo",
     "formatting": "run -T prettier --check ./src && run -T eslint ./src",
     "formatting:fix": "run -T prettier -w ./src",
-    "test": "NODE_NO_WARNINGS=1 node --experimental-vm-modules $(yarn bin jest) --passWithNoTests"
+    "test": "NODE_NO_WARNINGS=1 node --experimental-vm-modules $(yarn bin jest) --passWithNoTests --testTimeout=15000"
```

Comment:
I'd suggest adding the testTimeout to the specific tests that need it, rather to all of them. This will also let you keep the `inherits` from package.common. Still, I believe it's no longer needed after David's last changes (see [here](https://github.com/AztecProtocol/aztec3-packages/pull/263#discussion_r1166056038)).